### PR TITLE
Tests: option to run tests for only one config for each platform

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

**Comment by cpanato** [comment]
retest this please. now with azure and metal :)

**Comment by cpanato** [comment]
did not report back the results. but PR is green

File: Jenkinsfile [hunk 1]
```diff
@@ -92,6 +92,11 @@ pipeline {
       defaultValue: true,
       description: ''
     )
+    booleanParam(
+      name: 'BASIC_CLUSTER_CONFIG_ONLY',
+      defaultValue: true,
```
Comment (mxinden) [inline Jenkinsfile]: Default set to true would mean, that we have to edit all trigger jobs, right?

Comment (cpanato) [reply Jenkinsfile]: will set to false

Comment (cpanato) [reply Jenkinsfile]: done

**Comment by cpanato** [comment]
aws failed in the destroy phase because the credentials expired. I will create a PR to fix this issue

**Pull request closed without merging**